deploy/maistra-operator.yaml

```diff
@@ -3578,17 +3578,17 @@ spec:
         olm.relatedImage.v1_0.sidecar-injector: docker.io/maistra/sidecar-injector-ubi8:1.0.8
 
         olm.relatedImage.v1_1.3scale-istio-adapter: quay.io/3scale/3scale-istio-adapter:v1.0.0
-        olm.relatedImage.v1_1.citadel: docker.io/maistra/citadel-ubi8:1.1.3
-        olm.relatedImage.v1_1.cni: docker.io/maistra/istio-cni-ubi8:1.1.3
-        olm.relatedImage.v1_1.galley: docker.io/maistra/galley-ubi8:1.1.3
-        olm.relatedImage.v1_1.grafana: docker.io/maistra/grafana-ubi8:1.1.3
-        olm.relatedImage.v1_1.ior: docker.io/maistra/istio-ior-ubi8:1.1.3
-        olm.relatedImage.v1_1.mixer: docker.io/maistra/mixer-ubi8:1.1.3
-        olm.relatedImage.v1_1.pilot: docker.io/maistra/pilot-ubi8:1.1.3
-        olm.relatedImage.v1_1.prometheus: docker.io/maistra/prometheus-ubi8:1.1.3
-        olm.relatedImage.v1_1.proxy-init: docker.io/maistra/proxy-init-centos7:1.1.3
-        olm.relatedImage.v1_1.proxyv2: docker.io/maistra/proxyv2-ubi8:1.1.3
-        olm.relatedImage.v1_1.sidecar-injector: docker.io/maistra/sidecar-injector-ubi8:1.1.3
+        olm.relatedImage.v1_1.citadel: docker.io/maistra/citadel-ubi8:1.1.5
```

Comment:
I think these should all be 1.1.8 instead of 1.1.3.

Reply:
This matches what is in maistra-1.1 (community is at 1.1.5).  Also, we can bump that in the future, but for now, this PR blocks merging for all other 2.0 PRs (gen-check fails).

Reply:
We're about to ship 1.1.8.

Reply:
Community resolved in #538. I think this PR is ok.